opensrp-reveal/src/main/java/org/smartregister/reveal/widget/RevealEditTextFactory.java

```diff
@@ -47,6 +53,8 @@ protected void attachLayout(String stepName, Context context, JsonFormFragment f
         }
         editText.setGravity(Gravity.START);
         editText.setSingleLine(false);
+
+        addRelativeNumericIntegerValidator(jsonObject, formFragment, editText);
```

Comment:
Move this method call to native form EditTextFactory 
com.vijay.jsonwizard.widgets.EditTextFactory#attachLayout

Reply:
Done.